lib/index.js

```diff
@@ -54,17 +54,16 @@ internals.schema = Joi.object({
 .without('mapUri', 'port', 'protocol')
 .without('uri', 'port', 'protocol');
 
+exports.name = 'kibi-h2o2';
 
-exports.register = function (server, pluginOptions, next) {
+exports.register = function (server) {
 
-    server.handler('kibi_proxy', internals.handler);
+    server.decorate('handler', 'kibi_proxy', internals.handler);
 
-    server.decorate('reply', 'kibi_proxy', function (options) {
+    server.decorate('toolkit', 'kibi_proxy', function (options) {
 
         internals.handler(this.request.route, options)(this.request, this);
     });
-
-    return next();
```

Comment:
No need to return next() anyore?

Reply:
All the methods that used to require callbacks (e.g. `next()`) are now asynchronous. Return a promise and it is `await`ed instead.
![image](https://user-images.githubusercontent.com/7104356/99572926-4e158600-29cd-11eb-9eb2-a4923963db2d.png)